specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/machineLearningServices.json

```diff
@@ -2994,6 +3132,108 @@
       ],
       "x-ms-discriminator-value": "AmlCompute"
     },
+    "ComputeInstance": {
+      "description": "An Azure Machine Learning compute instance.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Compute"
+        },
+        {
```

Comment:
Could we please avoid multi-inheritance here?

Reply:
This inheritance is there from the beginning in stable spec. It will be a breaking change if I modify this.